setup.py

```diff
@@ -67,4 +70,7 @@
           # 'Programming Language :: Python :: 3.4',
       ],
       keywords="books ebooks gitenberg gutenberg epub metadata",
+      dependency_links=[
+          "https://github.com/gabalese/pyepub/archive/master.zip#egg=pyepub-0.3.1",
```

Comment:
this should be aligned with the package in requirements.pip

Reply:
Done